src/main/java/sisyphus/Parser.java

```diff
@@ -0,0 +1,165 @@
+package sisyphus;
```

Comment:
Well done on the overall code! Looks very clean

Though I think you can try to separate into more packages for example:
![image](https://github.com/nus-cs2103-AY2324S1/ip/assets/122634188/d1f0a1bc-1d93-433a-97b5-f10af9fa0e6d)

This will make the code more segmented and easier to follow in a large code base! :)